apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts

```diff
@@ -459,7 +459,7 @@ function AddGroupModal({ csrfToken }: { csrfToken: string }) {
           id="addGroupUids"
           name="uids"
           placeholder="student1@example.com, student2@example.com"
-          aria-describedby="addGroupUidsHelp"
+          aria-labelledby="addGroupUidsHelp"
```

Comment:
`aria-describedby` was correct here. This input already has a `<label>` associated with it. The help text is an auxiliary description, not the primary label.

Reply:
Reverted to `aria-describedby` in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).